app/controllers/application_controller.rb

```diff
@@ -65,10 +65,8 @@ def status
   #
   helper_method :sublayout
 
-  def redirect_back_or_to(*fallback_params)
-    redirect_to :back
-  rescue ActionController::RedirectBackError
-    redirect_to(*fallback_params)
+  def redirect_back_or_to(fallback_location)
```

Comment:
This is changing accepted parameters type. Can't break expectations somewhere?

Reply:
I checked the usage and every time the method was used we passed only a single parameter. So I don't think it will break anything.

Also, the tests passed